code/modules/mob/living/carbon/xenomorph/castes/defender/castedatum_defender.dm

```diff
@@ -84,15 +84,15 @@
 	upgrade_threshold = 200
 
 	// *** Defense *** //
-	armor = list("melee" = 25, "bullet" = 5, "laser" = 5, "energy" = 25, "bomb" = XENO_BOMB_RESIST_0, "bio" = 25, "rad" = 25, "fire" = 12, "acid" = 25)
+	armor = list("melee" = 25, "bullet" = 24, "laser" = 24, "energy" = 25, "bomb" = XENO_BOMB_RESIST_0, "bio" = 25, "rad" = 25, "fire" = 12, "acid" = 25)
 
 	// *** Defender Abilities *** //
-	crest_defense_armor = 40
-	fortify_armor = 80
+	crest_defense_armor = 26 //50 bullet/laser resistance
```

Comment:
dont put comments like this in here because they almost instantly become out of date.